src/app/lib/config/services/afrique.js

```diff
@@ -41,43 +41,44 @@ const service = {
     translations: {
       seeAll: 'Tout voir',
       home: 'Accueil',
-      currentPage: 'Current page',
+      currentPage: 'Page en cours',
       skipLinkText: 'Aller au contenu',
-      relatedContent: 'Related content',
+      relatedContent: 'Lire plus',
       error: {
         404: {
           statusCode: '404',
-          title: '404 - Page non trouvée',
+          title: 'La page ne peut pas être trouvée',
           message:
-            "Vous avez peut-être mal entré l'adresse internet. Veuillez vérifier l'adresse.",
+            "Désolé, nous ne pouvons pas afficher la page que vous cherchez. Essayez s'il vous plaît :",
           solutions: [
-            'Double checking the url',
-            'Hitting the refresh button in your browser',
-            'Searching for this page using the BBC search bar',
+            "Double vérification de l'url",
+            "Cliquez sur le bouton d'actualisation de votre navigateur",
+            "Recherche de cette page à l'aide de la barre de recherche de la BBC",
           ],
-          callToActionFirst: 'Alternatively, please visit the ',
-          callToActionLinkText: "Page d'accueil BBC",
+          callToActionFirst: 'Vous pouvez également consulter le site Web de ',
```

Comment:
Looks like this has been changed to `Vous pouvez également consulter le page d'accueil du site `